features/support/fs.rs

```diff
@@ -97,6 +101,10 @@ pub fn create_project() -> io::Result<Project> {
 }
 
 fn create_cargo_toml(dir: TempDir) -> io::Result<TempDir> {
+  fn double_backslash<T: Display>(path: T) -> String {
```

Comment:
You're telling me that the most specific type available is "Display?" That sounds weird, will take a look tonight. Its NBD, so if i cant fix it ill just merge this up.